pkgs/development/python-modules/ovirt-engine-sdk/default.nix

```
@@ -0,0 +1,37 @@
+{ 
+  lib,
+  buildPythonPackage,
```

most common style is to have the comma at the beginning, so whole lines can be added or more removed with out affecting other lines on diffs.
```suggestion
,  buildPythonPackage
```